uptraining_docai_processor_using_python/docai_uptraining.ipynb

```diff
@@ -0,0 +1,995 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.        dataset = documentai_v1beta3.Dataset(
</pre><p>initialize this using parameters instead of a dict, can be easier to read and check for type safety.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/741/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>